zaas-service/src/main/java/org/zowe/apiml/zaas/security/config/NewSecurityConfiguration.java

```diff
@@ -602,7 +601,7 @@ public WebSecurityCustomizer webSecurityCustomizer() {
                     // Endpoints that skip Spring Security completely
                     // There is no CORS filter on these endpoints. If you require CORS processing, use a defined filter chain
                     web.ignoring()
-                        .requestMatchers(InternalServerErrorController.ERROR_ENDPOINT, "/error",
+                        .requestMatchers("/error",
```

Comment:
It the contant removal correct? We have multiple similar endpoints, see `/error`, `/not_found`, and `/internal_error`.